lib/appenders/console.js

```diff
@@ -1,11 +1,17 @@
 "use strict";
 var layouts = require('../layouts')
-, consoleLog = console.log.bind(console);
+, levels = require('../levels')
+, consoleLog = console.log.bind(console)
+, consoleError = console.error.bind(console);
 
 function consoleAppender (layout, timezoneOffset) {
   layout = layout || layouts.colouredLayout;
   return function(loggingEvent) {
-    consoleLog(layout(loggingEvent, timezoneOffset));
+    if (loggingEvent.level.level >= levels.ERROR.level) {
```

Comment:
According to the node documentation, warnings are also logged to `stderr` since it's just an alias for `console.error`.
https://nodejs.org/api/console.html#console_console_warn_data